core/src/main/groovy/com/novoda/gradle/release/ReleasePlugin.groovy

```diff
@@ -19,6 +20,28 @@ class ReleasePlugin implements Plugin<Project> {
 
     void attachArtifacts(PublishExtension extension, Project project) {
         if (project.plugins.hasPlugin('com.android.library')) {
+            def current = GradleVersion.current()
+            def version3_3 = GradleVersion.version("3.3")
+
+            if ((current <=> version3_3) <= 0) {
+                // detected Gradle 3.3 or smaller
+                // show message for required gradle upgrade
+                // details: https://github.com/novoda/bintray-release/issues/112
+                System.err.println("""
```

Comment:
I'm not 100% sure but if we know that it will going ti fail anyways. Instead of printing, we can also do `throw new GradleException` with this message?